docs/guides/user-containers/readme.md

```diff
@@ -137,3 +137,63 @@ Now any user can select this profile in their Workflow by specifying it in a
 ```bash
 #DW container profile=red-rock-slushy  [...]
 ```
+
+### Using a Private Version of MPI File Utils
```

Comment:
I would title this Using Private Image Repositories. See my comment below.

Reply:
No, that heading is used above.  This new section is specifically about the adding the imagePullSecrets lists to the mpiSpec in the NnfContainerProfile.

Reply:
Whoops. I would then just title these sections as Adding Secrets for Non-MPI/MPI Containers or something to that affect.